galsim/detectors.py

```diff
@@ -297,7 +350,9 @@ def quantize(self):
     """
     self.applyNonlinearity(numpy.round)
 
+
 galsim.Image.applyNonlinearity = applyNonlinearity
 galsim.Image.addReciprocityFailure = addReciprocityFailure
```

Comment:
Not this code, but "add" also seems wrong to me for reciprocity failure.  Apply seems more apt to me here as well.  It doesn't add some kind of "reciprocity failure" feature to the image, it applies the effect of reciprocity failure to the ADU values in the image.